docs/source/dev-setup/devenv.rst

```diff
@@ -46,6 +46,129 @@ Prerequisites
 
     brew install jq
 
+Go Programming Language
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Hyperledger Fabric uses the Go Programming Language for many of its
+components.
+
+  - `Go <https://golang.org/dl/>`__ version 1.14.x is required.
+
+If you will be writing chaincode programs in Go, there are two
```

Comment:
This topic is for Fabric developers, not chaincode application developers.

Reply:
Removed that phrase since we are already on the page for Fabric developers.